### PR TITLE
update versions based on dependabot PRs

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: packages/synapse-react-client/package.json [hunk 1]
```diff
@@ -183,7 +183,6 @@
     "@types/react-window": "^1.8.5",
     "@types/shortid": "^0.0.29",
     "@types/spark-md5": "^3.0.2",
-    "@types/testing-library__jest-dom": "^5.14.7",
```
Comment (jay-hodgson) [inline packages/synapse-react-client/package.json:186]: Package deprecated (jest-dom now includes types): https://www.npmjs.com/package/@types/testing-library__jest-dom

**Comment by jay-hodgson** [comment]
As an extra step (besides verifying projects could build), I also pulled the SRC UMD build into synapse.org and it was loaded properly (using Chrome local override)

File: apps/SageAccountWeb/vite.config.ts [hunk 2]
```diff
@@ -5,5 +5,6 @@ export default mergeConfig(vitestConfig, {
   test: {
     include: ['src/tests/**/*.test.[jt]s?(x)'],
     setupFiles: ['src/tests/setupTests.ts'],
+    environment: 'jsdom', // introduced due to random "ReferenceError: window is not defined" during tests
```
Comment (hallieswan) [inline apps/SageAccountWeb/vite.config.ts:8]: Wonder why this popped up now! Looks like vite's [default environment](https://vitest.dev/config/#environment) is node

**Review by hallieswan — APPROVED** [review]
Looks good to me!

**Pull request merged**